docs/src/components/Example.js

```diff
@@ -10,80 +10,58 @@ type Props = {|
   description: string,
   name: string,
   scope: Object,
-  stacked: boolean,
-  stacked?: boolean,
```

Comment:
@chrislloyd this is my least favorite change - previously we were able to see the immediate changes with all examples, now (since they're always stacked) this doesn't work with certain examples: `FlyoutExample` / `TabExample` / `Tooltip`, we have to constantly scroll up / down to see the changes.